lightningd/params.c

```diff
@@ -269,10 +269,9 @@ static void check_params(struct param *params)
 }
 #endif
 
-static bool param_parse_arr(struct command *cmd,
-			    const char *buffer,
-			    const jsmntok_t tokens[],
-			    struct param *params)
+static bool param_arr(struct command *cmd, const char *buffer,
```

Comment:
Renaming a static was probably overkill :)

Reply:
Haha I know but it was a search and replace rename, and then I decided to keep it.